moveit_simple/src/moveit_simple.cpp

```diff
@@ -1206,14 +1243,26 @@ bool Robot::getFK(const std::vector<double> & joint_point,
                   Eigen::Affine3d &pose) const
 {
   virtual_robot_state_->setJointGroupPositions(joint_group_, joint_point);
-  const std::vector<std::string> link_names = joint_group_->getLinkModelNames();
-  const int vc =  (int) virtual_robot_state_->getVariableCount();
-  if ( joint_point.size() == vc)
+
+  if (ik_transforms_found_)
```

Comment:
If this were named `ik_transforms required`, would it make more sense.

Reply:
Don't need this anymore.